src/components/network/network-table.js

```diff
@@ -285,6 +900,45 @@ const NetworkTable = (props) => {
         );
     };
 
+    const getCSVFilename = useCallback(() => {
+        const tabName = TABLES_DEFINITION_INDEXES.get(tabIndex).name;
+        const localisedTabName = intl.formatMessage({ id: tabName });
+        return localisedTabName
+            .trim()
+            .replace(/[\\/:"*?<>|\s]/g, '-') // Removes the filesystem sensible characters
+            .substring(0, 27); // Best practice : limits the filename size to 31 characters (27+'.csv')
+    }, [intl, tabIndex]);
+
+    const getCSVColumnNames = () => {
+        let tempHeaders = [];
+        const columns = generateTableColumns(tabIndex).filter(
```

Comment:
Why filter out edit column ?

Reply:
If we don't filter it out, then it becomes an empty column in the CSV file.